# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/out/gen/angle/id/commit.h
# Commit: 040a6d3c401d
# Full Hash: 040a6d3c401de4142568a0c942f169244dc72a71
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-05-12 09:34:21
# Description:
#   Bug 1636122 - [angle] Use deviceContext instead of deviceContext1. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74770
# ==============================================================================

diff -r 9923799dd260 -r 040a6d3c401d gfx/angle/checkout/out/gen/angle/id/commit.h
--- a/gfx/angle/checkout/out/gen/angle/id/commit.h	Tue May 12 01:13:17 2020 +0000
+++ b/gfx/angle/checkout/out/gen/angle/id/commit.h	Tue May 12 01:22:38 2020 +0000
@@ -1,3 +1,3 @@
-#define ANGLE_COMMIT_HASH "8df54289d717"
+#define ANGLE_COMMIT_HASH "eabf2a79aac3"
 #define ANGLE_COMMIT_HASH_SIZE 12
-#define ANGLE_COMMIT_DATE "2020-05-06 11:04:42 -0700"
+#define ANGLE_COMMIT_DATE "2020-05-11 10:58:02 -0700"